CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 ## Unreleased
 
+The Spring Boot Starter (`opentelemetry-spring-boot-starter`) is now stable.
+
+### Migration notes
+
+- The Spring autoconfigure artifacts have been renamed
+  - `opentelemetry-spring-boot` &rarr; `opentelemetry-spring-boot-autoconfigure-2`
+  - `opentelemetry-spring-boot-3` &rarr; `opentelemetry-spring-boot-autoconfigure-3`
+- Two experimental HTTP metrics have been renamed:
+  - `http.server.request.size` &rarr; `http.server.request.body.size`,
+  - `http.server.response.size` &rarr; `http.server.response.body.size`
+
+### 📈 Enhancements
+
+- Support HTTP client instrumentation configuration in Spring starter
+  ([#11620](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11620))
+- Influxdb client: don't fill `db.statement` for create/drop database and write operations
+  ([#11557](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11557))
+- Add Javalin instrumentation
```

Comment:
It's a new instrumentation, put it into the part of New javaagent instrumentation like previous release note?
https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.4.0